crds/crd-coralogix-alert-set.yaml

```diff
@@ -78,6 +78,23 @@ spec:
                                 descprition: Sends resolve notification when the alert criteria that you defined is no longer met
                                 type: boolean
                                 default: true
+                              metricAlertPromqlParameters:
+                                description: Data for defining promql metric
+                                type: object
+                                properties:
+                                  promqlText:
```

Comment:
I think this one need to be in Filters, because its a different kind of Query (Now we have lucene and promQL query)